include/NovelRT/Ecs/Configurator.h

```diff
@@ -1,3 +1,4 @@
+#include "IEcsSystem.h"
```

Comment:
Please remove this, it has the potential to break due to how our headers are structured. If it's causing problems to remove I can probably walk you through how to fix it correctly.

Looks like there's an additional include here that should not exist and a missing namespace error preprocessor error we have in our headers, maybe we should fix that while we're here.

Reply:
Is it just a matter of including `Ecs.h` explicitely and remove the `#error`?

Reply:
Remove the include, add the error define unless it's already in Configurator.h and I missed it?

Reply:
Done. There is indeed a `#error` if the ECS define is not set, is that what you're expecting? The same can be found in 28 other files.

Reply:
Yeah I see it now.

There's still an include we should remove here but I'll flag it in the second pass. :)

Reply:
Ok nvm GitHub mobile won't let me - can you remove the `#include utility` at the top? The very first thing should be the file comment and there should be no additional includes in here.

Basically just fixing something that should not have been a problem in the first place.